scripts/processing/02-python_filtering.py

```diff
@@ -56,6 +57,11 @@ def run_filter(subject_id):
                    h_trans_bandwidth='auto', filter_length='auto',
                    phase='zero', fir_window='hann',
                    fir_design='firwin')
+        # high-pass eog channel to get reasonable thresholds in autoreject
+        picks_eog = pick_types(raw.info, meg=False, eog=True)
+        raw.filter(l_freq, None, picks=picks_eog, l_trans_bandwidth='auto',
+                   filter_length='auto', phase='zero', fir_window='hann',
+                   fir_design='firwin')
```

Comment:
let's do this later but if it's a key to make autoreject work out of the box we should do this internally in get_rejection_threshold